src/main/java/org/launchcode/liftoffrecipetracker/Models/User.java

```diff
@@ -0,0 +1,56 @@
+package org.launchcode.liftoffrecipetracker.Models;
+
+import javax.persistence.Entity;
+import java.util.ArrayList;
+
+@Entity
+public class User extends org.launchcode.liftoffrecipetracker.models.AbstractEntity {
+
+    private String username;
+    private String password;
+    private String email;
+    private ArrayList recipe = new ArrayList();
```

Comment:
For ArrayLists, kind of like with HashMaps, we need to include type parameters. It would look something like this: ArrayList<TypeGoesHere> variable = new ArrayList<>(); This just makes our code more secure and when we compile ensures that only the type we want to go in the ArrayList is there. If we were to create an ArrayList<String> and try to include an Integer with it, we would receive a compile error when we try to run the program.